package.json

```diff
@@ -1,8 +1,10 @@
 {
   "scripts": {
-    "test": "true"
+    "test": "true",
+    "preview": "netlify dev"
   },
   "devDependencies": {
-    "@11ty/eleventy": "^2.0.1"
+    "@11ty/eleventy": "^2.0.1",
+    "netlify-cli": "^16.9.1"
```

Comment:
We don't need the Netlify CLI, this repository is equipped with a dev container, and includes tasks that can be run.

Reply:
`.devcontainer.json` in the repo root appears to be for https://code.visualstudio.com/docs/devcontainers/create-dev-container, is that true? Does this work with _anything_ other than Visual Studio Code?

Reply:
Sure, devcontainers are an open standard; for example, all Intellj editors support it, and even neovim supports that stuff nowadays. Besides that, it works out of the box in GitHub Codespaces, which you can access directly here from our repositories.

Reply:
> Sure, devcontainers are an open standard; for example, all Intellj editors support it, and even neovim supports that stuff nowadays. Besides that, it works out of the box in GitHub Codespaces, which you can access directly here from our repositories.

Ah, my  apologies. I only skimmed that Microsoft page since I was on mobile and it really didn't make it obvious.